app/models/mixins/miq_policy_mixin.rb

```diff
@@ -29,7 +29,7 @@ def get_policies(mode = nil)
       klass, id = t.split("/")
       next unless ["miq_policy", "miq_policy_set"].include?(klass)
       policy = klass.camelize.constantize.find_by(:id => id.to_i)
-      mode.nil? || policy.mode == mode ? policy : nil
+      mode.nil? || policy if policy.mode == mode
```

Comment:
Is this one actually different, and happens to works out, or is this equivalent.  In this case, the `if` has precedence for the whole line, but I'm not sure about the ternary.  In either case, I'd use `()` to disambiguate.

Reply:
redid